lib/hawkular/operations/operations_api.rb

```diff
@@ -105,7 +127,8 @@ def connect
         client.on(:message, once: true) do |msg|
           parsed_message = msg.data.to_msg_hash
 
-          logger.log("Sent WebSocket message: #{parsed_message}")
+          # Not working with websocket-vcr
```

Comment:
Ok, logging is back, and session_id is removed.
Looks like session_id has been broken for some time. But we didn't realized that something was wrong. Notice that logging is before setting the session_id. The out-of-scope variable was causing an exception but ruby throws it silently because the exception happens in a secondary thread. So, indeed, the line setting the session_id was never being reached.